ci/cpu/upload.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Adopted from https://github.com/tmcdonell/travis-scripts/blob/dfaac280ac2082cd6bcaba3217428347899f2975/update-accelerate-buildbot.sh
```

Comment:
This comment can probably be removed.

Reply:
Ok. Will remove. This just came from [here]( https://github.com/rapidsai/cusignal/blob/600eb63d48ecb21ef4a374bbb91a0f6fa34b4828/ci/cpu/upload.sh#L3 )

Reply:
Removed